components/brave_wallet_ui/components/desktop/views/portfolio/components/asset-more-popup/index.tsx

```diff
@@ -0,0 +1,51 @@
+// Copyright (c) 2022 The Brave Authors. All rights reserved.
```

Comment:
Can we rename this file to `asset-more-popup.tsx`?

Reply:
Fixed here: f0f6f66ffb7984f3a43025eb001034ee77d394e8